### DISCUSSION TITLE
package name should be normalized

### DISCUSSION THREAD

**Discussion by superiums** [void-linux/void-packages#46844] [General]
i think its better to have a rule for all pkg names.

for example, we should use `-` other than `_` `,` `:` to split words.
we should use `thunar-dev` other than `thunarDev`.

**i think we should never use UPPERCASE char in pkg name.**
just use `thunar` instead of `Thunar`, `waybar` instead of `Waybar`.
we don't have any good reason to use UPPERCASE char in pkg name, it will leads to complex and puzzle. hard to search it out. hard to remove if typed error.

Do you agree ?

**Comment by classabbyamp** [comment]
> i think its better to have a rule for all pkg names.

we already have [rules for package names](https://github.com/void-linux/void-packages/blob/master/Manual.md#namingconventions)

changing existing package names is not trivial because it will lead to a mass of transitional dummy packages for little benefit.

case insensitivity for xbps has been discussed before: https://github.com/void-linux/xbps/issues/506

**Comment by ahesford** [comment]
It's not "hard to search it out", because `xbps-query` is case-insensitive. It might be harder to "guess it out" with `xbps-install`, but installing packages by guess when you aren't positive they are what you want isn't great practice.
